contrib/!source-control/github/packages.el

```diff
@@ -86,3 +87,25 @@
         "gfC" 'spacemacs/git-link-commit-copy-url-only)
       ;; default is to open the generated link
       (setq git-link-open-in-browser t))))
+
+(defun github/init-magit-gh-pulls ()
+  (use-package magit-gh-pulls
+    :commands magit-gh-pulls-mode
```

Comment:
Isn't this line redundant? When using the melpa version, `magit-gh-pulls-mode` is already autoloaded.